src/common/base/Base.h

```diff
@@ -135,6 +135,20 @@ struct PartMeta {
     }
 };
 
+struct SpaceMeta {
```

Comment:
Do we really need this structure. 
It is only used in meta client,   it seems not so necessary.

Reply:
Besides that,  do you prepare to cache the charset information in meta client.

Reply:
> necessary
When multiple parameters are passed, we can pass a structure variable.

Reply:
> Besides that, do you prepare to cache the charset information in meta client.
Good catch.   I will do it in the following  pr.

Reply:
If so, use another name for it.  For example,  SpaceDesc.  And put it in MetaClient.h

Reply:
Good suggestion.